src/ebay-button/__tests__/index.stories.tsx

```diff
@@ -99,3 +99,11 @@ storiesOf(`ebay-button`, module)
             <p><EbayButton priority="primary" bodyState="loading">Primary Button</EbayButton></p>
         </>
     ))
+    .add(`Open in new tab/window`, () => (
+        <EbayButton>
```

Comment:
Why don't we add the link and the link target props? Otherwise, it's just a button with label, not even a link.